File: judge/runners/templates/java/EvaluatorExecutor.java
## Responsible for generating a function call to a custom evaluator.
<%page args="evaluator,expected,actual,arguments" />
import java.util.*;

public class EvaluatorExecutor {

    public static void main(String[] args) throws Exception {
        try (AbstractCustomEvaluator eval = new ${evaluator}()) {
            eval.evaluate(
                <%include file="value.mako" args="value=expected"/>,
                <%include file="value.mako" args="value=actual"/>,
                <%include file="value.mako" args="value=arguments"/>
            );
        }
    }
}

File: judge/src/tested/languages/haskell/templates/EvaluatorExecutor.hs
<%page args="evaluator,expected,actual,arguments,output" />
module EvaluatorExecutor where

import qualified ${evaluator}
import Values


main = ${evaluator}.evaluate
                "${output}",
                <%include file="value.mako" args="value=expected"/>,
                <%include file="value.mako" args="value=actual"/>,
                <%include file="value.mako" args="value=arguments"/>
